ffi/src/primitives.rs

```diff
@@ -169,16 +171,43 @@ impl<'itm> IntoFfiSliceRef<'itm> for core::cmp::Ordering {
     }
 }
 
+/// Trait for replacing unsupported types with supported ones when crossing FFI-boundary (for example in case of wasm)
+pub trait PrimitiveRepr {
```

Comment:
```suggestion
#[cfg(feature = "wasm")]
pub trait WasmRepr {
```

why not this naming?

Reply:
Ok, this naming, will be more precise.
Initially i choose `PrimitiveRepr` because i want to minimize places where we have conditional compilation.